pkg/modprobe/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 as mirror
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    musl \
+    && true
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+RUN mkdir /out/bin && mkdir /out/sbin && cp /bin/busybox /out/bin/busybox
+
+FROM scratch
+ENTRYPOINT []
+CMD []
+WORKDIR /
+COPY --from=mirror /out/ /
+RUN ["/bin/busybox", "ln", "-s", "/bin/busybox", "/sbin/modprobe"]
+LABEL org.mobyproject.config='{"binds": ["/lib/modules:/lib/modules", "/sys:/sys"], "capabilities": ["CAP_SYS_MODULE"]}'
```

Comment:
I did not need the bind of `/sys:/sys` in my testing, but I found that the following tests included it in their `binds`, so I included `/sys:/sys` in the `LABEL` for `pkg/modprobe` -

https://github.com/linuxkit/linuxkit/blob/3b097e6056b7b0b9396c7908c82d72212813a847/test/cases/040_packages/005_extend/001_btrfs/test.yml#L13

https://github.com/linuxkit/linuxkit/blob/3b097e6056b7b0b9396c7908c82d72212813a847/test/cases/040_packages/005_extend/001_btrfs/test-create.yml#L13

https://github.com/linuxkit/linuxkit/blob/3b097e6056b7b0b9396c7908c82d72212813a847/test/cases/040_packages/006_format_mount/003_btrfs/test.yml#L13